.github/workflows/main.yml

```diff
@@ -228,11 +228,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        scenario: [ember-release, ember-beta, ember-canary]
+        scenario: [ember-release, ember-beta]
```

Comment:
canary is timing out.  If canary is failing now, beta will soon as well.  Tests just aren't starting up.

Reply:
I think we talked about potentially excluding this.  But perhaps I'm misremembering what that was.

Reply:
Yeah that makes sense, we probably want to not have `beta` block CI either.